templates/lib/prism/serialize.rb.erb

```diff
@@ -44,6 +44,9 @@ module Prism
     end
 
     class Loader # :nodoc:
+      USE_SIMPLE_STRINGIO = RUBY_ENGINE == "truffleruby"
```

Comment:
Can you just make this a class?

```suggestion
      IO = RUBY_ENGINE == "truffleruby" ? SimpleStringIO : StringIO
```

Reply:
Done, I named it `FastStringIO` to not "shadow" ::IO and convey more that's it's a StringIO-like class specifically.